File: src/main/java/com/distribution/modules/sys/dao/DistrictDao.java
package com.distribution.modules.sys.dao;

import com.distribution.modules.sys.entity.District;
import org.apache.ibatis.annotations.Mapper;

import java.util.List;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.sys.dao
 * @Description TODO(描述)
 * @create 2018/6/20-21:58
 */
@Mapper
public interface DistrictDao {

    List<District> getByParentId(String parentId);

}

File: src/main/kotlin/com/distribution/modules/sys/dao/DistrictDao.kt
package com.distribution.modules.sys.dao

import com.distribution.modules.sys.entity.District
import org.apache.ibatis.annotations.Mapper

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.sys.dao
 * @Description TODO(描述)
 * @create 2018/6/20-21:58
 */
@Mapper
interface DistrictDao {

    fun getByParentId(parentId: String): List<District>

}
